### DISCUSSION TITLE
How do I classify my resources in the Swagger UI renderer?

### DISCUSSION THREAD

**Discussion by jtech237** [api-platform/api-platform#2603] [Q&A]
Here's my problem:
I've defined my entities as Api platform resources, and now in the documentation they're in an order I don't really like.
How can I put them in the order I want?

**Comment by ERuban** [comment]
You can use swagger tags to group operations https://swagger.io/docs/specification/grouping-operations-with-tags/
For more complex changes you always can to decorate api-platform's OpenApi normalizer and reorder your operations in `$documentation['paths']`
